Makefile

```diff
@@ -43,9 +43,12 @@ else
 endif
 
 .PHONY: ui
-ui:
+ui: sematic/ui/node_modules/.build_timestamp
```

Comment:
My intuition tells me the change to this makefile was prompted by an out-of-date `package_lock.json`. I have also deployed main today, and it was out of date.

Doesn't this PR have to come with an updated `package_lock.json`?

Reply:
Discussed offline. Thanks!